README.md

```diff
@@ -50,7 +50,7 @@ Run this script before raising a PR to ensure your code changes pass the Jenkins
 Start up service in SM2 as shown above then:
 
 ```
-../run_local_register_beneficiaries.sh
+./run_local_register_beneficiaries.sh
```

Comment:
This file doesn't seem to exist any more in _trusts-acceptance_tests_ so we probably need to change this.  There's `./run_suite.sh RegisterBeneficiariesRunner` which might be the equivalent.